components/script/dom/bluetooth.rs

```diff
@@ -443,6 +443,18 @@ impl BluetoothMethods for Bluetooth {
         return p;
     }
 
+    #[allow(unrooted_must_root)]
```

Comment:
Because our rooting lint doesn't understand that `Rc<Promise>` is safe. I thought there was an issue about it, but I can't find it.